go/signedexchange/signedexchange.go

```diff
@@ -97,6 +99,13 @@ func (e *Exchange) encodeRequest(enc *cbor.Encoder) error {
 			valueE.EncodeByteString([]byte(e.requestUri.String()))
 		}),
 	}
+	for name, value := range e.requestHeaders {
```

Comment:
encodeRequest from L172 should include requestHeaders, while L149 should not

Reply:
Done.